v3/plugins/eclipse/che-theia/latest/meta.yaml

```diff
@@ -48,7 +48,7 @@ spec:
         discoverable: false
   containers:
    - name: theia-ide
-     image: "docker.io/eclipse/che-theia:7.2.0"
+     image: "docker.io/eclipse/che-theia:7.3.0"
```

Comment:
Latest has a different `firstPublishedDate` than `7.3.0`

Reply:
but it is expected, no?

Reply:
for me 'latest' = first release of 'latest', not sure if it makes much sense though